pkg/runner/runner.go

```diff
@@ -140,15 +140,20 @@ func (r *JMeterRunner) Run(execution testkube.Execution) (result testkube.Execut
 
 	jtlPath := filepath.Join(outputDir, "report.jtl")
 	reportPath := filepath.Join(outputDir, "report")
-	args := []string{"-n", "-t", path, "-l", jtlPath, "-e", "-o", reportPath}
+	jmeterLogPath := filepath.Join(outputDir, "jmeter.log")
+	args := []string{"-n", "-j", jmeterLogPath, "-t", path, "-l", jtlPath, "-e", "-o", reportPath}
 	args = append(args, params...)
 
 	// append args from execution
 	args = append(args, execution.Args...)
 	output.PrintLog(fmt.Sprintf("%s Using arguments: %v", ui.IconWorld, args))
 
+	mainCmd := "jmeter"
+	if key := os.Getenv("ENTRYPOINT_CMD"); key != "" {
```

Comment:
LookupEnv?

Reply:
I think Getenv is fine, because I am only interested is the value != "", I don't care is it defined empty or not defined at all